src/style.css

```diff
@@ -2,2 +1,73 @@
+body,
+ul,
+ol,
+li {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is correctly loaded in your HTML file. If not, the fallback to 'sans-serif' will occur.